.azure-pipelines/generate-beta-modules-template.yml

```diff
@@ -0,0 +1,262 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This file should be renamed to generate-module-template.yml. It generates both v1.0 and beta modules.